packages/grid/x-data-grid/src/tests/columnHeaders.DataGrid.test.tsx

```diff
@@ -49,4 +49,49 @@ describe('<DataGrid /> - Column Headers', () => {
       expect(getColumnHeaderCell(0)).to.have.class('foobar');
     });
   });
+
+  describe('header menu', () => {
+    it('should allow to hide column', () => {
+      const { getByRole, getAllByLabelText } = render(
```

Comment:
Prefer using the queries from `screen`. https://kentcdodds.com/blog/common-mistakes-with-react-testing-library#not-using-screen We still have some tests written with bad practices.